test_scenarios/kubernetes/3idx1sh1cm-pvc/splunk-master-pvc.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
@matthewmodestino As discussed, this might/should/will(?) cause issues when using a multiple AZ deployment on AWS. I was going to suggest that you create a set of AWS specific pvcs, where you call out the zone specifically, but it looks like they're deprecating that

> From: https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs
> - `zone` (Deprecated): AWS zone. If neither zone nor zones is specified, volumes are generally round-robin-ed across all active zones where Kubernetes cluster has a node. zone and zones parameters must not be used at the same time.
> - `zones` (Deprecated): A comma separated list of AWS zone(s). If neither zone nor zones is specified, volumes are generally round-robin-ed across all active zones where Kubernetes cluster has a node. zone and zones parameters must not be used at the same time.

Unless they make the deployment smart enough to shift to the zone of the pvc choosen, then the only other option I can see is to create an `indexer-master` StatefulSet` with a replication setting of 1.